airgun/entities/contenthost.py

```diff
@@ -55,12 +59,44 @@ def execute_package_action(self, entity_name, action_type, value):
         view.progressbar.wait_for_result()
         return view.read()
 
+    def execute_module_stream_action(self, entity_name, action_type, module_name, stream_version):
```

Comment:
From product point of view: does it make sense to provide some default value to some of these arguments? Or do all have to be specified every time?

Reply:
We can not pass the default value, as module stream content itself is built over different yum repos. There are chances if someone writing test from yum repo does contain same default module stream.

Reply:
I can think of the only action_type, other are dynamic in nature.

Reply:
OK, that's fair. Just wanted confirmation.